govcd/vapp.go

```diff
@@ -35,15 +35,21 @@ func (vcdCli *VCDClient) NewVApp(client *Client) VApp {
 
 // struct type used to pass information for vApp network creation
 type VappNetworkSettings struct {
-	Name             string
-	Gateway          string
-	NetMask          string
-	DNS1             string
-	DNS2             string
-	DNSSuffix        string
-	GuestVLANAllowed *bool
-	StaticIPRanges   []*types.IPRange
-	DhcpSettings     *DhcpSettings
+	Id                 string
```

Comment:
I think we are using capital cased ID strings in types.

```suggestion
	ID                 string
```

Reply:
changed